src/features/SaveAndCloseButtons/components/SaveAndCloseButtons.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import propTypes from "prop-types";
+import { Button } from "carbon-components-react";
+import { FormattedMessage } from "react-intl";
+import "../styles/SaveAndCloseButtons.scss";
+
+export const SaveAndCloseButtons = (props) => {
```

Comment:
SaveAndCloseButtons are not feature, its a common component so it should sit inside src/component folder